packages/salesforcedx-vscode-core/src/orgBrowser/nodeTypes.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as vscode from 'vscode';
+
+export enum NodeType {
+  Org,
+  MetadataType,
+  MetadataCmp
+}
+
+export class Node extends vscode.TreeItem {
```

Comment:
Another nit - "Node" seems generic when this is specifically for the org browser. Maybe MetadataNode or OrgBrowserNode or something.